vi_minimal.ipynb

```diff
@@ -7,7 +7,7 @@
         "colab_type": "text"
```

Comment:
<p>Per our discussion, could you set the recall target here to None?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf-isoscapes/pull/207/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>